spec/jobs/upvs/fetch_services_with_forms_list_job_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe Upvs::FetchServicesWithFormsListJob, type: :job do
+  describe '#perform' do
+    let(:url) { 'https://example.com' }
+
+    let(:downloader) { double }
+
+    it 'downloads and imports ServicesWithForms list in V1 format' do
+      expect(downloader).to receive(:download_file).with(url).and_return(fixture_filepath('upvs/services-v1.csv'))
+
+      subject.perform(url, downloader: downloader)
+
+      expect(Upvs::ServiceWithForm.first).to have_attributes(
+        instance_id: 2107,
+        external_code: 'App.GeneralAgenda',
+        meta_is_code: 'NULL',
+        name: 'Veobecná agenda',
+        type: 'Formulárové sluby',
+        uri: 'ico://sk/30797764',
+        institution_name: 'Agentúra na podporu výskumu a vývoja',
+        url: 'https://schranka.slovensko.sk/FormConstructor/Default.aspx?IdService=2107',
+        info_url: 'NULL',
+        form_url: 'http://schemas.gov.sk/form/App.GeneralAgenda/1.9'
```

Comment:
@smith558 ked kukam na tie hodnoty, toto mi tiez pride ako gulas v nazvoch... ale najskor, co byva v `info_url` okrem `nil`?

Reply:
`'sluzba_egov_20'`

Reply:
@smith558 v `info_url` je `sluzba_egov_20`? fuha...

Reply:
Jaj, prepac, **chyba**. Myslel som `meta_is_code` ktory je tiez `nil` niekedy.

`info_url` vyzera ze je nil na vsetkych zaznamoch. Zatial.

Reply:
@smith558 oka, no podla mna tu treba mat pripady kde to nie je `nil`, presiel si to uz cele a uplne vsade je to `nil`?

Reply:
Okay, v 6310 zo 213000 zaznamoch sa nachadza aj non-`nil` `info_url`
napr.

`https://www.dcom.sk/detail-sluzby?nazov=poskytovanie-navratnej-financnej-vypomoci`
`http://esluzby.dubnica.eu/Sluzby/Sluzba/28`
`http://esluzby.dubnica.eu/Sluzby/31`

Reply:
- dobroš, nazov `info_url` je ok
- nevies nahodou co je na https://schranka.slovensko.sk/FormConstructor/Default.aspx?IdService=2107 ?

lebo toto celkom nesedi `form_url` ->  `'http://schemas.gov.sk/form/App.GeneralAgenda/1.9'` to je skor `form_schema_url` ale treba zistit najprv co je na https://schranka.slovensko.sk/FormConstructor/Default.aspx?IdService=2107

Reply:
Na tu url sa ja neviem dostat, nemam pristup na SK.sk. Ty nemas nejaky pristup? @pavolzbell 

Originalne na _data.gov.sk_ to bol tiez `FormURL` , takze neviem? Ale pozeram ze mnozstvo (niekolko 1000) zaznamov ma `form_url` `nil`

Reply:
@smith558 ok, `nil` prec a dajme `form_url` -> `schema_url` a je to hotove

Reply:
Okay, vyhadzujem `null: false `, `form_url` -> `schema_url`